PowerEditor/src/Notepad_plus.rc

```diff
@@ -705,6 +705,17 @@ BEGIN
             MENUITEM SEPARATOR
             MENUITEM "Move Tab Forward",        IDM_VIEW_TAB_MOVEFORWARD
             MENUITEM "Move Tab Backward",       IDM_VIEW_TAB_MOVEBACKWARD
+            MENUITEM SEPARATOR
+            POPUP "Colour"
+            BEGIN
+                MENUITEM "Red",      IDM_VIEW_TAB_COLOUR_1
+                MENUITEM "Green",    IDM_VIEW_TAB_COLOUR_2
+                MENUITEM "Blue",     IDM_VIEW_TAB_COLOUR_3
+                MENUITEM "Orange",   IDM_VIEW_TAB_COLOUR_4
+                MENUITEM "Purple",   IDM_VIEW_TAB_COLOUR_5
+                MENUITEM SEPARATOR
+                MENUITEM "None",     IDM_VIEW_TAB_COLOUR_NONE
+            END

```

Comment:
1. For the sake of usability, the 2nd level of sub-menu should be avoided.
2. The visual colour would be better than word, and it could be customizable by users.

So I would do : 
```
       POPUP "Tab"
        BEGIN
            MENUITEM "1st Tab",                 IDM_VIEW_TAB1
            MENUITEM "2nd Tab",                 IDM_VIEW_TAB2
            MENUITEM "3rd Tab",                 IDM_VIEW_TAB3
            MENUITEM "4th Tab",                 IDM_VIEW_TAB4
            MENUITEM "5th Tab",                 IDM_VIEW_TAB5
            MENUITEM "6th Tab",                 IDM_VIEW_TAB6
            MENUITEM "7th Tab",                 IDM_VIEW_TAB7
            MENUITEM "8th Tab",                 IDM_VIEW_TAB8
            MENUITEM "9th Tab",                 IDM_VIEW_TAB9
            MENUITEM SEPARATOR
            MENUITEM "Next Tab",                IDM_VIEW_TAB_NEXT
            MENUITEM "Previous Tab",            IDM_VIEW_TAB_PREV
            MENUITEM SEPARATOR
            MENUITEM "Move Tab Forward",        IDM_VIEW_TAB_MOVEFORWARD
            MENUITEM "Move Tab Backward",       IDM_VIEW_TAB_MOVEBACKWARD
            MENUITEM SEPARATOR
            MENUITEM "Apply Color 1",      IDM_VIEW_TAB_COLOUR_1
            MENUITEM "Apply Color 2",    IDM_VIEW_TAB_COLOUR_2
            MENUITEM "Apply Color 3",     IDM_VIEW_TAB_COLOUR_3
            MENUITEM "Apply Color 4",   IDM_VIEW_TAB_COLOUR_4
            MENUITEM "Apply Color 5",   IDM_VIEW_TAB_COLOUR_5
            MENUITEM "Remove Color",     IDM_VIEW_TAB_COLOUR_NONE
        END
```